devel/standard/standard.yaml

```diff
@@ -11,7 +11,7 @@ start:
     mindreader-node-debug-deep-mind: true
     mindreader-node-log-to-zap: true
     # Indicates to the mindreader-node to skip writing individual one block files and merge the block data into 100-blocks merged files
-    mindreader-node-merge-and-store-directly: false
+    mindreader-node-merge-and-store-directly: true
     # Flags that will be added to the dummy chain process command
     mindreader-node-arguments: +--dm-enabled --block-rate=6
```

Comment:
This line should be removed, it unused in arweave case

Reply:
great! I hate this line